src/client/server.rs

```diff
@@ -58,6 +58,7 @@ async fn solve(r: web::Json<GeetestResponse>, appdata: web::Data<AppData>) -> im
         phone: appdata.phone.clone(),
         sec_code: r.geetest_seccode.clone(),
         validation_code: r.geetest_validate.clone(),
+        channel: None,
```

Comment:
Is "SMS" the default?

Reply:
Yes, `SMS` is indeed the default value for channel.
Refer to [this line of code](https://github.com/GaloyMoney/galoy/blob/af4fa719d62ada4ea812695b24f7ee1816ee7aad/src/graphql/root/mutation/captcha-request-auth-code.ts#LL61C1-L61C1)